scripts/migratrion-0.5.0/partitioned/1-add-indices-manually.sql

```diff
@@ -0,0 +1,37 @@
+DO $$
+DECLARE
+    v_rec record;
+    v_parent_table constant text := 'journal';
```

Comment:
Schema, table and column names are customisable. 

Consider wrapping this block within a procedure which takes table & column names as a parameter. 

Take a look on old scripts: https://github.com/SwissBorg/akka-persistence-postgres/tree/v0.1.0/scripts/partitioned/migration - in our case such procedure would need to take at least 3 parameters: one for parent table name and other two for persistence_id and sequence_number column names.

Reply:
I chose a slightly different way, extracting variables and listing them at the top level without creating a procedure